apps/frontend/src/app/5_pages/BitocracyPage/components/NewProposalForm/NewProposalForm.constants.ts

```diff
@@ -0,0 +1,82 @@
+import { ProposalType } from './NewProposalForm.types';
+
+export const proposalTypeOptions = [
+  {
+    value: ProposalType.Parameter,
+    label: ProposalType.Parameter,
+  },
+  {
+    value: ProposalType.Proclamation,
+    label: ProposalType.Proclamation,
+  },
+  {
+    value: ProposalType.Treasury,
+    label: ProposalType.Treasury,
+  },
+];
+
+export const proposalContractOptions = [
+  {
+    value: 'Sovryn Protocol',
+    label: 'Sovryn Protocol',
+  },
+  {
+    value: 'Staking',
+    label: 'Staking',
+  },
+  {
+    value: 'LoanTokenLogicLM',
+    label: 'LoanTokenLogicLM',
+  },
+  {
+    value: 'LoanTokenLogicWRBTC',
+    label: 'LoanTokenLogicWRBTC',
+  },
+  {
+    value: 'iDOC',
+    label: 'iDOC',
+  },
+  {
+    value: 'iUSDT',
+    label: 'iUSDT',
+  },
+  {
+    value: 'iRBTC',
+    label: 'iRBTC',
+  },
+  {
+    value: 'iBPRO',
+    label: 'iBPRO',
+  },
+  {
+    value: 'iXUSD',
+    label: 'iXUSD',
+  },
+];
+
+export const proposalTreasuryOptions = [
+  {
+    value: 'Sovryn Protocol',
```

Comment:
I think this should be an enum as well because you'll want to compare it somewhere and you don't want to have magic strings.